### DISCUSSION TITLE
Multiple Screens Function Mashed

### DISCUSSION THREAD

**Discussion by nick-theboatman** [lxqt/lxqt#2234] [Q&A]
I've just upgraded from Ubuntu 18.04 + LXDE to Ubuntu 20.04 + LXQt and been having the usual probs getting the system up BUT I have encountered a show-stopper that means that we cant use Zoom (whatever) which is a major trauma. 

Just to clarify, we used the screen on the laptop ("left-hand screen") for file management and other stuff, a HDMI wide-screen telly ("right-hand screen") for the Zoom session. Laptop (Tosh Sat C50) resolution is 1366 x768, the telly (Tosh Regza) is 1920 x 1080. The laptop desktop also uses a photo as the wallpaper which is 1280 x 800 set to "fill screen". This all worked fine with Ubuntu 18.04 and LXDE but has fallen apart with Ubuntu 20.04 and LXQt. With the telly powered up and connected and trying to set the resolution has been very problematic in that no matter which combination of resolutions we tried we couldn't get anything that worked. 

2-SCREENS MODE

In particular the left hand screen bleeds over to the right-hand screen (by about 50%) and the bottom panel also appears half-way up the right-hand screen extending to the right by about 50%; the right-hand third of the wallpaper picture appears on the right hand screen and there is a vertical white line that appears to be the boundary between the left desktop and the right desktop. The wallpaper image that falls onto the  right hand screen goes right to the bottom of the screen thus the bottom panel cuts across the middle of it. Only the left half of the right hand desktop is displayed on the right hand screen, presumably the rest has fallen off the end of the world. This includes the maximise / restore buttons top right :/ It was not clear in the screen config that the numbers to fill in in the resolution settings are in fact not resolution values but coordinates settings for the top-left corners respective screens. It would be advisable to put in a label to say this otherwise and unsuspecting user (that's me) might not clock this. 

IDENTICAL SCREENS MODE

With both screens displaying the same content ("clones") they were very not identical and not usable. In particular the loss of the maximise/ restore buttons (top right of any window) fall out of sight and are thus not recoverable - the bootstrap way of getting them back is to use the screen configurator to change the coordinates of the screens to swap them round and thus regain visibility of the maximise / restore / close buttons.

So all-in-all a real mess but I don't know where to stat... 
One thing that I noticed was that the resolution choice was pre-fixed to a small number of options in a pull-down. I though that laptop LCD controllers (and certainly the telly) had multiple resolution settings that could be selected by messing with the graphics controller (and in the case of the telly using the telly set-up in real time). 
Any pointers to start this one along-the-way?

**Comment by tsujan** [comment]
You may want to report the problem you've encountered to Ubuntu. The LXQt version that Ubuntu 20.04 has should be very old.

If the problem happens with a recent release (≥ 1.0.0), please report it at https://github.com/lxqt/lxqt/issues by following the issue template.

**Reply by stefonarch** [reply]
You can upgrade to the latest LXQt release using this ppa:
https://launchpad.net/~severusseptimius/+archive/ubuntu/lxqt
